src/components/previewers/ImagePreviewer.jsx

```
@@ -0,0 +1,150 @@
+
+            {totalComments > 0 && (
+              <span className="action-button">
+                {totalComments} {totalComments === 1 ? "comment" : "comments"}
+              </span>
+            )}
+
+            <button
+              onClick={handleCopy}
+              className={`action-button hover:${textColor}`}
+              aria-label="Copy Link"
+            >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)